### PR TITLE
Adds Faucet information for Demo EVM AppChain

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

Adds info on the Demo EVM Appchain Faucet

### Checklist

- [x] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [ ] If this page requires a disclaimer, I have added one
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects

### CODE REVIEW

File: builders/tanssi-network/networks/dancebox/demo-evm-containerchain.md [hunk 1]
```diff
@@ -25,6 +25,23 @@ The demo EVM ContainerChain HTTPS and WSS endpoints are as follows:
     wss://fraa-dancebox-3001-rpc.a.dancebox.tanssi.network/
     ```
 
+## Faucet {: #faucet }
+
+You can access TANGO tokens at the faucet at [apps.tanssi.network/demo](https://apps.tanssi.network/demo){target=\_blank}. You can receive up to 100 TANGO tokens every 12 hours. Please do not spam the faucet with unnecessary requests. To request tokens from the faucet, head to [apps.tanssi.network/demo](https://apps.tanssi.network/demo){target=\_blank} and Press **Add to MetaMask**.
```
Comment (eshaben) [inline builders/tanssi-network/networks/dancebox/demo-evm-containerchain.md:30]:
```suggestion
You can access TANGO tokens at the faucet on the [Tanssi dApp](https://apps.tanssi.network/demo){target=\_blank}. You can receive up to 100 TANGO tokens every 12 hours. Please do not spam the faucet with unnecessary requests. To request tokens from the faucet, head to the [Tanssi dApp](https://apps.tanssi.network/demo){target=\_blank} and Press **Add to MetaMask**.
```

This actually came up in a recent report, but using the link as the link text is not considered descriptive, so just saying something like "Tanssi dApp" would pass that check

File: builders/tanssi-network/networks/dancebox/demo-evm-containerchain.md [hunk 2]
```diff
@@ -25,6 +25,23 @@ The demo EVM ContainerChain HTTPS and WSS endpoints are as follows:
...
+
+![Add to MetaMask](/images/builders/network/networks/dancebox/demo/demo-1.webp)
+
```
Comment (eshaben) [inline builders/tanssi-network/networks/dancebox/demo-evm-containerchain.md:33]:
```suggestion
```

File: builders/tanssi-network/networks/dancebox/demo-evm-containerchain.md (hunk 1 above)
Comment (eshaben) [reply builders/tanssi-network/networks/dancebox/demo-evm-containerchain.md:30]: you could also say "Tanssi faucet" here... or whatever you think is best!

**Review by augustocollado — DISMISSED** [review]
LGTM! 
Maybe we should add a note stating that the TANGO tokens are meant for development and testing purposes and don't have any real value?

**Pull request merged**